Firmware/mmu2/progress_codes.h

```diff
@@ -13,7 +13,7 @@ enum class ProgressCode : uint_fast8_t {
     UnloadingToFinda, // P3
     UnloadingToPulley, //P4
     FeedingToFinda, // P5
-    FeedingToBondtech, // P6
+    FeedingToExtruder, // P6
```

Comment:
we should rename the state in the MMU FW - progress_codes.h and error_codes.h are normally copied from the MMU repo